documentation/src/main/asciidoc/topics/proc_troubleshooting_xsite.adoc

```diff
@@ -0,0 +1,53 @@
+[id='xsite_tshoot-{context}']
+= Collecting Logs and Troubleshooting Cross-Site Replication
+Diagnose and resolve issues related to {brandname} cross-site replication. Use
+the {brandname} Command Line Interface (CLI) to adjust log levels at run-time
+and perform cross-site troubleshooting.
+
+.Procedure
+
+. Open a terminal in `{server_home}`.
+. Create a {brandname} CLI connection.
+. Adjust run-time logging levels to capture DEBUG or TRACE messages if necessary.
++
+For example, the following command enables TRACE log messages for the org.infinispan.XSITE category:
++
+[source,options="nowrap",subs=attributes+]
+----
+[//containers/default]> logging set --level=TRACE org.infinispan.XSITE
```

Comment:
we don't allow TRACE in runtime (without restarting the server). You can enable DEBUG.

Reply:
updated